src/react-components/src/components/Icons/IconTypes.tsx

```diff
@@ -0,0 +1,6 @@
+export enum iconRotationOptions {
```

Comment:
Don't really know where to put this since it's merged but uhhh 
1) I think, since it's `Types`, let's call this enum `iconRotationTypes` to differentiate it from the `-Options` objects (Which means something different in the Design System context) 
2) Since this is something consumers need access to, remember to export it in `index.ts`